### PR TITLE
Update client card (DEV-86)

### PR DESCRIPTION
**Pull request by mikefeldberg**
DEV-86

contact info will be added in DEV-166, so we can't show residence address working here. Below screenshot is how residence address will appear once implemented

Screenshot 2024-09-27 at 12 15 17

### CODE REVIEW

File: libs/expo/shared/ui-components/src/lib/ClientCard/ClientCard.tsx [hunk 1]
```diff
@@ -1,24 +1,31 @@
...
+  IdCardOutlineIcon,
   LocationDotIcon,
   TentIcon,
   ThreeDotIcon,
+  UserOutlineIcon,
 } from '@monorepo/expo/shared/icons';
 import { Colors, Radiuses, Spacings } from '@monorepo/expo/shared/static';
+import { format } from 'date-fns';
 import { useRouter } from 'expo-router';
+import { HmisProfileType } from 'libs/expo/betterangels/src/lib/apollo/graphql/__generated__/types';
```
Comment (mikefeldberg) [inline libs/expo/shared/ui-components/src/lib/ClientCard/ClientCard.tsx:11]: @Davit-BetterAngels this import is incorrect and I'm not sure what it needs to be. Could you please take a look and update as needed?

File: libs/expo/betterangels/src/lib/ui-components/ClientCard.tsx [hunk 2]
```diff
@@ -1,24 +1,33 @@
...
-import IconButton from '../IconButton';
-import TextBold from '../TextBold';
-import TextButton from '../TextButton';
-import TextRegular from '../TextRegular';
-
+import { HmisProfileType } from '../apollo';
 type TSpacing = 'xs' | 'sm' | 'md' | 'lg' | 'xl';
-
 interface IClientCardProps {
   imageUrl?: string;
```
Comment (Davit-BetterAngels) [inline libs/expo/betterangels/src/lib/ui-components/ClientCard.tsx:22]: as we moved under betterangels path can we type from generated types? 


besides progress and arrivedFrom everything can be ClientProfile I guess

File: libs/expo/shared/ui-components/tsconfig.lib.json [hunk 3]
```diff
@@ -14,5 +14,11 @@
     "**/*.stories.tsx"
   ],
   "files": ["../../../../node_modules/@nx/expo/typings/svg.d.ts"],
-  "include": ["**/*.js", "**/*.jsx", "**/*.ts", "**/*.tsx"]
+  "include": [
+    "**/*.js",
+    "**/*.jsx",
+    "**/*.ts",
+    "**/*.tsx",
+    "../../betterangels/src/lib/ui-components/ClientCard.tsx"
```
Comment (Davit-BetterAngels) [inline libs/expo/shared/ui-components/tsconfig.lib.json:22]: what is this for ?

**Review by Davit-BetterAngels — CHANGES_REQUESTED** [review]

File: libs/expo/shared/ui-components/tsconfig.lib.json (hunk 3 above)
Comment (mikefeldberg) [reply libs/expo/shared/ui-components/tsconfig.lib.json]: I think this got added automagically by vscode. removed

File: libs/expo/betterangels/src/lib/ui-components/ClientCard.tsx (hunk 2 above)
Comment (mikefeldberg) [reply libs/expo/betterangels/src/lib/ui-components/ClientCard.tsx]: good call. updated

**Review by Davit-BetterAngels — APPROVED** [review]

File: libs/expo/betterangels/src/lib/screens/Clients/Clients.graphql [hunk 4]
```diff
@@ -37,13 +37,16 @@ query ClientProfiles(
   clientProfiles(filters: $filters, pagination: $pagination, order: $order) {
     ... on ClientProfileType {
       id
+      age
```
Comment (shiozakiyuko) [inline libs/expo/betterangels/src/lib/screens/Clients/Clients.graphql:40]: Hey @mikefeldberg 

2 things I noticed.

1. Is the Last Location coming later on?
2. The HMIS ID is not matching on the card and profile (personal Info).

Comment (mikefeldberg) [reply libs/expo/betterangels/src/lib/screens/Clients/Clients.graphql:40]: 1. yes. Residence address will be added in DEV-166. Conditional location of last interaction will be added in DEV-760. 
2. fixed

Comment (shiozakiyuko) [reply libs/expo/betterangels/src/lib/screens/Clients/Clients.graphql:40]: I see your fix now. :)

**Review by shiozakiyuko — APPROVED** [review]

**Pull request merged**